### PR TITLE
add manual mark as dirty

### PR DESCRIPTION
**Pull request by ChristianMoll**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser-Support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser-Support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

File: src/lib/components/forms/form/form-entry-list/terra-form-entry-list.component.ts [hunk 1]
```diff
@@ -126,6 +126,7 @@ export class TerraFormEntryListComponent implements OnInit, OnChanges, ControlVa
             this.formArray.push(
                 TerraFormHelper.createNewControl(this.inputFormField.defaultValue, this.inputFormField)
             );
+            this.inputFormGroup.markAsDirty();
```
Comment (pweyrich) [inline src/lib/components/forms/form/form-entry-list/terra-form-entry-list.component.ts:129]: Ich glaube du solltest eher das formArray as dirty markieren. Das wird sich zwar wahrscheinlich auch auf den parent (die `inputFormGroup`) auswirken, aber ich denke das wäre das richtige control.

Comment (MFrank84) [reply src/lib/components/forms/form/form-entry-list/terra-form-entry-list.component.ts:129]: Wenn das Array `dirty`sein soll, dann sollte auch das Array als solches markiert werden. Status wie dirty, valid usw. werden in einer FormGroup immer nach oben gebubbelt 🙂

Comment (ChristianMoll) [reply src/lib/components/forms/form/form-entry-list/terra-form-entry-list.component.ts]: formArray funktioniert 👍

**Review by MFrank84 — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 5.X.X (xx.xx.xxxx)
```
Comment (pweyrich) [inline CHANGELOG.md:1]: Bitte hier die Version eintragen, die aus deiner Änderung resultiert.

**Review by MFrank84 — APPROVED** [review]

**Comment by felixdausch** [comment]
✅  Im ShopBuilder getestet

**Review by pweyrich — APPROVED** [review]

**Pull request merged**